Adds lazy loading guidance to Optimize LCP guide. (#8123)
GoogleChrome/web.dev: src/site/content/en/blog/optimize-lcp/index.md

Here are some examples where the LCP resource cannot be discovered from scanning the HTML document response:

- The LCP element is an `<img>` that is dynamically added to the page via JavaScript.
- The LCP element is lazily loaded with a JavaScript library that hides its `src` or `srcset` attributes (often as `data-src` or `data-srcset`).
- The LCP element requires a CSS background image.

> In both each of these cases, the browser needs to run the script or apply the stylesheet—which usually involves waiting for network requests to finish—before it can discover the LCP resource and could start loading it. This is never optimal.

Reviewer: s/both each/each/

#### Optimize the priority the resource is given

In some cases, the LCP resource is discoverable from the HTML markup, but it _still_ doesn't start loading as early as the first resource. This can happen if the browser preload scanner's priority heuristics do not recognize that the resource is important, or if it determines that other resources are more important.

> For example, you can deprioritize your LCP image via HTML if you set [`loading="lazy"`](/browser-level-image-lazy-loading/) on your `<img>` element. The reduced priority means that the resource will receive less bandwidth, thus potentially incurring a loading delay.

Reviewer: The resource may receive less bandwidth, but I think more likely the resource would just start loading later, which is the primary issue I see in the wild.

> For example, you can deprioritize your LCP image via HTML if you set [`loading="lazy"`](/browser-level-image-lazy-loading/) on your `<img>` element. The reduced priority means that the resource will receive less bandwidth, thus potentially incurring a loading delay.

Author: ```suggestion
For example, you can deprioritize your LCP image via HTML if you set [`loading="lazy"`](/browser-level-image-lazy-loading/) on your `<img>` element. The reduced priority means that the resource may begin loading later than it otherwise would.
```

Beyond avoiding patterns such as specifying `loading="lazy"` on your LCP image, you can hint to the browser as to which resources are most important via the <code>[fetchpriority](/priority-hints/)</code> attribute for resources that could benefit from a higher priority:

```html
<img fetchpriority="high" src="/path/to/hero-image.webp">
```

It's a good idea to set `fetchpriority="high"` on an `<img>` element if you think it's likely to be your page's LCP element—but limit this to just one or two images (based on common desktop and mobile viewport sizes), otherwise the signal becomes meaningless. You can also lower the priority of images that may be early in the document response, but isn't visible due to styling, such as images in carousel slides that are not visible at startup:

```html
<img fetchpriority="low" src="/path/to/carousel-slide-3.webp">
```

Deprioritizing certain resources can afford more bandwidth to resources that need it more—but be careful. Always check resource priority in DevTools and test changes with lab and field tools.

{% Aside 'warning' %}

> Never lazy-load your LCP image, as that will always lead to unnecessary resource load delay and/or prioritization, and will have a negative impact on LCP.

Reviewer: Deprioritization still manifests as "resource load delay" (the LCP sub-part), so I'd remove the "or prioritization" part here.

> Never lazy-load your LCP image, as that will always lead to unnecessary resource load delay and/or prioritization, and will have a negative impact on LCP.

Author: ```suggestion
Never lazy-load your LCP image, as that will always lead to unnecessary resource load delay, and will have a negative impact on LCP.
```
